### PR TITLE
Feature/add migration doc

### PR DESCRIPTION
**Pull request by leborgneromain**

### CODE REVIEW

File: migrator-tool/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## Run Spline Migration from 0.3 to 0.4+
+
+In order to migrate the date from mongodb to arangodb you can use Spline Migrator Tool. 
+
+-   To list the options you can run the command:  `java -jar migrator-tool.jar --help` 
+
+```
```
Comment (wajda) [inline migrator-tool/README.md]: Codacy Issue found: [[fenced-code-flag] Missing code-language flag](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4623041)

File: migrator-tool/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  -c, --continuous         Watch the source database and migrate the incoming data on the fly
+  -l, --log-level <value>  Log level (`ALL`, `TRACE`, `DEBUG`, `INFO`, `WARN`, `ERROR`, `OFF`). Default is `ERROR`
+  --help                   prints this usage text
+
+```
+
+-   Example: `java -jar migrator-tool/target/migrator-tool.jar --source=mongodb://localhost:27017/splinedb --target=http://localhost:8080/spline/producer`
+
+
+---
```
Comment (wajda) [inline migrator-tool/README.md]: Codacy Issue found: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4623041)

File: migrator-tool/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    Copyright 2019 ABSA Group Limited
```
Comment (wajda) [inline migrator-tool/README.md]: Codacy Issue found: [[code-block-style] Code blocks should be fenced](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4623041)

File: README.md [hunk 4]
```diff
@@ -195,6 +195,12 @@ You also need to set some configuration properties. Spline combine these propert
 #### `spline.producer.url`
 -   url of spline producer (part of rest gateway responsible for storing lineages in database)
 
+##Run Spline Migration from 0.3 to 0.4+
+
+Spline 0.3 was running with mongoDB. In Spline 0.4 ArangoDB was introduced. To migrate the data from mongoDB to ArangoDB, Simply run: `java -jar migrator-tool/target/migrator-tool.jar --source=mongodb://localhost:27017/splinedb --target=http://localhost:8080/spline/producer`
```
Comment (wajda) [inline README.md]: is `--argument=value` syntax working? Shouldn't it be space separated `--argument value`?

**Comment by wajda** [comment]
I'd also mention that it's not a DB-to-DB kind of migration, but rather importing data from old an Mongo to a new pline via the Producer service. So the one must be running (with reference to a guide how to spin up the REST gateway)

File: README.md (hunk 4 above)
Comment (leborgneromain) [reply README.md]: both work

File: migrator-tool/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+The old data will be transformed and send to the Producer Service. Thus the first step is to startup the rest-gateway. 
+Please find the documentation [here](https://github.com/AbsaOSS/spline#start-spline-server) for this step.
+Once it is done, you can run the migration using this example below:
+
+-   Example: `java -jar migrator-tool/target/migrator-tool.jar --source=mongodb://localhost:27017/splinedb --target=http://localhost:8080/spline/producer`
+
+
+---
+
+    Copyright 2019 ABSA Group Limited
```
Comment (wajda) [inline migrator-tool/README.md]: Codacy Issue found: [[code-block-style] Code blocks should be fenced](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4623041)

File: migrator-tool/README.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
+## Run Spline Migration from 0.3 to 0.4+
+
+In order to migrate the date from mongodb to arangodb you can use Spline Migrator Tool. 
+
+-   To list the options you can run the command:  `java -jar migrator-tool.jar --help` 
+
+```
```
Comment (wajda) [inline migrator-tool/README.md]: Codacy Issue found: [[fenced-code-flag] Missing code-language flag](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4623041)

File: migrator-tool/README.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+The old data will be transformed and send to the Producer Service. Thus the first step is to startup the rest-gateway. 
+Please find the documentation [here](https://github.com/AbsaOSS/spline#start-spline-server) for this step.
+Once it is done, you can run the migration using this example below:
+
+-   Example: `java -jar migrator-tool/target/migrator-tool.jar --source=mongodb://localhost:27017/splinedb --target=http://localhost:8080/spline/producer`
+
+
+---
```
Comment (wajda) [inline migrator-tool/README.md]: Codacy Issue found: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4623041)

**Pull request merged**